_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "What DevOps Looks Like In Terms Of Our Process"
+ref: what-devops-looks-like
+author: "Omar Nasr, in collaboration with the SDS and the IT Strategy team"
+lang: en 
+date: "2020-02-16"
+last_modified: "2020-02-16"
+excerpt_separator: <!--more-->
+---
+
+<!--markdownlint-disable MD033-->
+
+In case you haven't noticed, there has been a large shift in our job as the technology organization of ESDC. We are now, more than ever, required to respond exponential change in technology and requirements faster, cheaper and with high quality. Gone are the days where we can plan for projects 5-10 years in advance. We've seen it over and over that by the time these projects are completed we have expended significant resources to produce a product that no longer meets our clients needs and are forced to spend even more to catch up to changed requirements. Continuing with this operating model means we are taking on significant risk in not being able to respond to the increasing social change and needs of our citizens. Fortunately, we are not a unique case, many technology organizations around the world are facing similar issues in both the public and private domain. A lot of organizations have responded to this by adopting DevOps methodologies, techniques, tools and processes. You may have heard your co-worker or manager talking about the need to apply DevOps principles. However, to address the elephant in the room, how can this possibly achieved with the large amount of processes currently in place. What does DevOps actually look like in terms of our organization? More specifically, how can we incorporate these principles in our release process to be able to enable frequent releases, automated testing and checks and be able to quickly respond to changes. What work needs to be done to shift to this model?
```

Comment:
In case you haven't noticed, there has been a large shift in our job as the technology organization of ESDC.

==>

There has been a large shift in our job as the technology organization of ESDC.

-----

Shorter, more to the point, less words, and less likely to come off as a belitting